README.md

```diff
@@ -1,2 +1,579 @@
 # Itaas Node.js tools
 
+## List of contents
+---------------------
+   
+ * [Introduction](#introduction)
+ * [Developers! Read it before everything explodes](#developers-read-it-before-everything-explodes)
+ * [API Tools](#api-tools)
+   * [createLogger](#createlogger)
+   * [createCallContext](#createcallcontext)
+   * [createServiceLocator](#createservicelocator)
+   * [createFieldSelector](#createfieldselector)
+   * [createBaseResponse](#createbaseresponse)
+   * [createFixedTimeService](#createfixedtimeservice)
+   * [createCurrentTimeService](#createcurrenttimeservice)
+   * [number](#number)
+   * [uuid](#uuid)
+   * [date](#date)
+   * [express.createCallContextMiddleware](#expresscreatecallcontextmiddleware)
+   * [express.createMorganMiddleware](#expresscreatemorganmiddleware)
+   * [express.createLowercaseQueryMiddleware](#expresscreatelowercasequerymiddleware)
+   * [promise.any](#promiseany)
+   * [cassandra.cql](#cassandracql)
+   * [cassandra.createBatchQueryBuilder](#cassandracreatebatchquerybuilder)
+   * [cassandra.converter.map](#cassandraconvertermap)
+   * [cassandra.converter.uuid](#cassandraconverteruuid)
+
+---------------------
+
+## Introduction
+This module should be used in ITaaS projects in order to simplify common things:
+
+You should only install and use it.
+```bash
+$ npm install https://github.com/UUX-Brasil/itaas-nodejs-tools.git#master --save
+```
+
+## Developers! Read it before everything explodes 
+
+This library have some rules:
+- **Freedom**: Everyone is free to pull request
+- **Dependencies**: Only really important things should be here. Don't use it to add more dependencies to your project. Useless dependencies are not fun!
+- **Test**: You also are responsible for testing everything. Currently we have 100% of code coverage. Do not think to PR here if coverage will be less than it 
+- **Documentation**: Keep the documentation updated and in English
+
+To contribute, clone this project and procedure with known commands:
+```bash
+$ npm clone (this github repository URL)
+$ cd itaas-nodejs-tools
+$ npm install
+```
+
+Before run tests, there is a file which configures where is your cassandra. "(itaas-nodejs-tools folder)/test/integration".
+If you are in a linux machine and it has docker, you can just run the following command:
```

Comment:
Change to if you have docker...
Docker 1.12 in Windows will run this command the same way in Windows and Linux.